clients/unshaded/src/main/java/tachyon/hadoop/AbstractTFS.java

```diff
@@ -55,7 +55,7 @@
  * All implementing classes must define {@link #isZookeeperMode()} which states if fault tolerant is
  * used and {@link #getScheme()} for Hadoop's {@link java.util.ServiceLoader} support.
  */
-abstract class AbstractTFS extends FileSystem {
+public abstract class AbstractTFS extends FileSystem {
```

Comment:
Because otherwise using {@link Foo} from outside of the package won't work. I am happy to keep the visibility the same and instead replace {@link ...} with just Foo.